test/format/ops.c

```diff
@@ -372,11 +375,11 @@ snap_check(WT_CURSOR *cursor,
 				return (ret);
 
 		/* Check for simple matches. */
-		if (ret == 0 && !start->deleted &&
+		if (ret == 0 && start->vset &&
```

Comment:
This (and the others) has become less readable with `vset` instead of `deleted`. What about checking `start->op != REMOVE`?

Reply:
> This (and the others) has become less readable with vset instead of deleted. What about checking `start->op != REMOVE`?

Pushed in 187fdd217.